### ISSUE TITLE
Add italics support for codios

### ISSUE THREAD

**Issue by vinayak-mehta** [vinayak-mehta/present#41]

**Comment by pstuerner** [comment]
Hi, can I pick this up?

**Comment by vinayak-mehta** [comment]
Yes! Thanks :)

**Comment by vinayak-mehta** [comment]
This now depends on #59.

**Comment by pstuerner** [comment]
This might be a bigger issue. Changing `codio.yaml` to this

```
speed: 10
lines:
- prompt: $
  in: normal
  color: green
  normal: true
- prompt: $
  in: bold
  color: green
  bold: true
- prompt: $
  in: underline
  color: green
  underline: true
- prompt: $
  in: reverse
  color: green
  reverse: true
```

and running `present codio.md` on my Windows machine (cmd terminal), gives me the following:

![grafik](https://user-images.githubusercontent.com/41574499/91826615-1dc81200-ec3e-11ea-8344-206c565718c6.png)

Underline and reverse have no effect.

Furthermore, my console cannot encode the `codio.yaml` progress bar character (█):

![grafik](https://user-images.githubusercontent.com/41574499/91826971-8e6f2e80-ec3e-11ea-90cc-d635bd800b66.png)

Probably a separate issue. OS compatibility could be a major task.

**Comment by vinayak-mehta** [comment]
> Underline and reverse have no effect.

Will have to look into which Windows terminals are supported better.

> Furthermore, my console cannot encode the codio.yaml progress bar character (█):
> Probably a separate issue. OS compatibility could be a major task.

Yeah, I think we should also change the default progress character to a `#`.
